resources/js/admin/settings/components/SettingsMenuCollapse.vue

```diff
@@ -90,6 +95,25 @@ export default {
         element.groups = this.orderGroupAlphabetic(element.groups);
       });
     },
+    checkChangeEmailServer() {
```

Comment:
A better solution can be implemented.
e.g. pass a variable to the component and directly select a new mail handler is added.